packages/valory/connections/p2p_libp2p/connection.py

```diff
@@ -525,9 +518,8 @@ async def start(self) -> None:
                         )
                     )
             else:  # pragma: nocover
-                self.logger.error(
-                    "Please check log file {} for more details.".format(self.log_file)
-                )
+                log_data = Path(self.log_file).read_text()
+                self.logger.error(f"Failure to connect to Libp2pNode:\n{log_data}")
```

Comment:
this is very useful to see, otherwise the file is deleted as soon as the temporary directory is cleaned. It contains information from the libp2p_node (golang) module informing you of the underlying issue.